projects/fhi-angular-components/src/lib/fhi-tree-view-selection/fhi-tree-view-selection.component.ts

```diff
@@ -24,6 +24,9 @@ export class FhiTreeViewSelectionComponent {
   @Output() itemsChange = new EventEmitter<Item[]>();
 
   ngOnInit() {
+    if (this.items === undefined) {
```

Comment:
I think this is unnecessary, the test in ngOnChanges() is enough.

Reply:
Without this test I get the following error:
![image](https://github.com/folkehelseinstituttet/Fhi.Frontend.Demo/assets/8872303/7a36ebb7-b3b0-4ee8-b84e-d54727355329)

Reply:
Aha, the test should be in the template then. Better than generatin dummy data.

Reply:
Also remove initial assignment, since we now accept undefined (@Input() items: Item[];)

Reply:
When test is done in the template, the test on OnChanges is not nescessary anymore, I guess?...